doc/ansible/dev/ansible_operator_sdk_developer_guide.md

```diff
@@ -141,7 +141,7 @@ default       Active    28d
 kube-public   Active    28d
 kube-system   Active    28d
 ```
-## Using Ansible inside of an Operator
+## Using an Ansible Operator
```

Comment:
@chris-short This might be semantics but I think this still needs to be titled `Using Ansible inside of an Operator` since this section outlines how to map and trigger an Ansible role in the operator.

@jmrodri @fabianvf can correct me if it's fine either way.

Reply:
We use `Ansible Operator` everywhere else as far as I can tell and it is the same thing. For consistency's sake can we change it?

Reply:
Hi @chris-short the title `Ansible Operator SDK Developer Guide` will be indexed. If we change this sub-title it will not lead the correct idea for the users. Could you please just revert this point then I think we will be able to move forward with. WDYT?

Reply:
I assume that by the time a user gets to this part of the documentation, they will have a decent idea of what we mean by "Ansible Operator". +1 to the patch.